packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -224,9 +224,17 @@ export class LitDevSearch extends LitElement {
           Cancel
         </button>
       </div>
-      <ul id="items" role="listbox" class=${classMap({expanded: items.length})}>
-        ${this._renderGroups()}
-      </ul>
+      ${this._searchText == '' || items.length > 0
+        ? html`
+            <ul
+              id="items"
+              role="listbox"
+              class=${classMap({expanded: items.length})}
+            >
+              ${this._renderGroups()}
+            </ul>
+          `
+        : html`<div id="no-items">no results</div>`}
```

Comment:
Can you please link to a new bug report on this repo with something prefilled like:

```
Subject: No search results for `${this.searchText}`

Content: <!-- What type of content did you expect to see on lit.dev and explain why it should be on lit.dev -->
```

Here's a nifty article on how to generate a URL for prefilled github issues:

https://eric.blog/2016/01/08/prefilling-github-issues/

Reply:
Pseudocode here could be:


```ts
html`<a href=${this.getGithubIssueUrl(this.searchText)} target="blank">No results</a>`
...
getGithubIssueUrl(searchText: string) {
  const githubIssueUrl = new URL('https://github.com/lit/lit.dev/issues/new');
  githubIssueUrl.searchParams
      .append('title', `[docs] No search results for \`${searchText}\``);
  githubIssueUrl.searchParams
      .append('body', `<!-- What type of content did you expect to see on lit.dev and explain why it should be on lit.dev -->`);
  return githubIssueUrl.href;
}
```

Reply:
Is it ok if I write this function in `url-helper.ts` in `util` ?